docs/_docs/sql-reference/ddl.adoc

```diff
@@ -84,28 +114,58 @@ CREATE TABLE IF NOT EXISTS Person (
   age int,
   company varchar,
   PRIMARY KEY (id, city_id)
-) WITH "template=partitioned,backups=1,affinity_key=city_id, key_type=PersonKey, value_type=MyPerson";
+) WITH "template=partitioned,backups=1,affinity_key=city_id,CACHE_NAME=Person,KEY_TYPE=PersonKey,VALUE_TYPE=PersonValue";
 ----
 
-Once the CREATE TABLE command gets executed, the following happens:
 
-- A new distributed cache is created and named SQL_PUBLIC_PERSON. This cache stores objects of the `Person` type that corresponds to a specific Java, .NET, C++ class or BinaryObject. Furthermore, the key type (`PersonKey`) and value type (`MyPerson`) are defined explicitly assuming the data is to be processed by key-value and other non-SQL APIs.
-- SQL table/schema with all the parameters will be defined.
-- The data will be stored in the form of key-value pairs. The `PRIMARY KEY` columns will be used as the object's key; the rest of the columns will belong to the value.
-- Distributed cache related parameters are passed in the `WITH` clause of the statement. If the `WITH` clause is omitted, then the cache will be created with default parameters set in the CacheConfiguration object.
+=== CREATE TABLE Using non-UpperCase Columns Example
+
+Both of Ignite’s SQL engines (H2 and Calcite) all unquoted identifiers, names of a table columns convert to upper case
```

Comment:
Here we shouldn't refer to engines (H2 and Calcite), something like 'Ignite's SQL parser converts all unquted identifiers to upper case', I think, will be better.